### PR TITLE
ax_python_module[_version]: Add Python 3 support.

### PR DESCRIPTION
**Pull request by Kurt-von-Laven**
Permit dependency checking of both Python 2 and Python 3 modules.

### CODE REVIEW

File: m4/ax_python_module.m4 [hunk 1]
```diff
@@ -21,13 +23,13 @@
...
 
-#serial 6
+#serial 7
 
 AU_ALIAS([AC_PYTHON_MODULE], [AX_PYTHON_MODULE])
 AC_DEFUN([AX_PYTHON_MODULE],[
     if test -z $PYTHON;
     then
-        PYTHON="python"
+        PYTHON="$3"
```
Comment (peti) [inline m4/ax_python_module.m4]: What happens here when `$3` is unset?

Comment (Kurt-von-Laven) [inline m4/ax_python_module.m4]: Currently that will confuse this code into thinking that the specified Python module can't be found. Would you prefer that we give a more helpful error message in this case saying too few parameters were passed? Alternatively, we could default the third parameter to "python."

Comment (peti) [inline m4/ax_python_module.m4]: Well, the synopsis suggests that the parameter is optional, so IMHO the macros should do something useful even if the 3rd parameter is left unspecified.

Comment (Kurt-von-Laven) [inline m4/ax_python_module.m4]: Doh; I didn't mean to mark the parameter optional, but it seems like a good idea to me. Now the third parameter defaults to "python3," but I'm definitely open to other suggestions.

**Comment by Kurt-von-Laven** [comment]
I see that GitHub hid my reply, so just wanted to let you know that I believe I have addressed your comment.

**Comment by peti** [comment]
Very cool. Thank you for your efforts!

**Comment by Kurt-von-Laven** [comment]
Thank you!

**Pull request merged**